tests/glm/test_glm.py

```diff
@@ -1788,3 +1789,49 @@ def test_score_method(as_data_frame, offset, weighted):
 
     # use pytest because NumPy used to always reject comparisons against zero
     assert pytest.approx(score, 1e-8) == int(offset is not None)
+
+
+def test_information_criteria(regression_data):
+    X, y = regression_data
+    regressor = GeneralizedLinearRegressor(family="normal", alpha=0)
+    regressor.fit(X, y)
+    assert np.allclose(
+        [138.80, 141.59, 168.20],
```

Comment:
Are these values taken from a previous run?
If possible, I would test the results against statsmodels.

Reply:
They are taken from a statsmodels run on the same data. I can include a note there that they are from statsmodels or just include the statsmodels fit in the test?